ssz/sedes/serializable.py

```diff
@@ -89,9 +92,9 @@ def merge_args_to_kwargs(args, kwargs, arg_names):
 
 
 class BaseSerializable(collections.Sequence):
-    _cached_ssz = None
+    db = None
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, db=None, *args, **kwargs):
```

Comment:
I suspect it would be better to put the `db` argument after `*args` so that it's required to be passed in as a keyword argument.